assets/src/domain/blocks/event-attendees/controls/SelectEvent.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { SelectControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+import { AttendeesEditProps } from '../types';
+import useEvents from '../hooks/useEvents';
+
+interface SelectEventProps {
+	event: string;
+	setEvent?: (event: string) => void;
+}
+
+const SelectEvent: React.FC<AttendeesEditProps> = ({ attributes, setAttributes }) => {
```

Comment:
let's also move this into the same folder as the `SelectDatetime` control
because a dropdown control for selecting events will be required in nearly every other block we ever build

Reply:
no comment or action on this?